libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -46,7 +66,9 @@ export class VisionExplanationDashboard extends React.Component<
 
     this.state = {
       imageDim: 200,
+      pageSize: 10,
```

Comment:
nit: maybe get this value from the available PageSizeOptions here, to keep it in sync with the list of options